name,value,relevant log,explanation
io.bytes.per.checksum,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
seq.io.sort.factor,100,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",property-item-hit	io
ha.failover-controller.new-active.rpc-timeout.ms,60000,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",property-item-hit	ha
dfs.ha.tail-edits.namenode-retries,3,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
ha.failover-controller.cli-check.rpc-timeout.ms,20000,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",property-item-hit	ha
io.bytes.per.checksum,0,"ERROR	Task: attempt_1701312551309_0590_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
seq.io.sort.factor,100,"ERROR	Task: attempt_1701312551309_0590_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	io
ha.failover-controller.new-active.rpc-timeout.ms,60000,"ERROR	Task: attempt_1701312551309_0590_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	ha
dfs.ha.tail-edits.namenode-retries,3,"ERROR	Task: attempt_1701312551309_0590_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
ha.failover-controller.cli-check.rpc-timeout.ms,20000,"ERROR	Task: attempt_1701312551309_0590_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	ha
io.bytes.per.checksum,0,"INFO	Diagnostics report from attempt_1701312551309_0590_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
seq.io.sort.factor,100,"INFO	Diagnostics report from attempt_1701312551309_0590_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	io
ha.failover-controller.new-active.rpc-timeout.ms,60000,"INFO	Diagnostics report from attempt_1701312551309_0590_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	ha
dfs.ha.tail-edits.namenode-retries,3,"INFO	Diagnostics report from attempt_1701312551309_0590_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
ha.failover-controller.cli-check.rpc-timeout.ms,20000,"INFO	Diagnostics report from attempt_1701312551309_0590_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",property-item-hit	ha
io.bytes.per.checksum,0,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
seq.io.sort.factor,100,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
ha.failover-controller.new-active.rpc-timeout.ms,60000,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	ha
dfs.ha.tail-edits.namenode-retries,3,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	ha
ha.failover-controller.cli-check.rpc-timeout.ms,20000,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	ha
io.bytes.per.checksum,0,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
seq.io.sort.factor,100,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
ha.failover-controller.new-active.rpc-timeout.ms,60000,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	ha
dfs.ha.tail-edits.namenode-retries,3,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	ha
ha.failover-controller.cli-check.rpc-timeout.ms,20000,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	ha
io.bytes.per.checksum,0,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
seq.io.sort.factor,100,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,0,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
seq.io.sort.factor,100,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
